master/internal/core_trial.go

```diff
@@ -2,24 +2,12 @@ package internal
 
 import (
 	"github.com/labstack/echo/v4"
-
-	"github.com/determined-ai/determined/master/internal/api"
 )
 
 func (m *Master) getTrial(c echo.Context) (interface{}, error) {
 	return m.db.RawQuery("get_trial", c.Param("trial_id"))
 }
 
-func (m *Master) getTrialDetails(c echo.Context) (interface{}, error) {
-	args := struct {
-		TrialID int `path:"trial_id"`
-	}{}
-	if err := api.BindArgs(&args, c); err != nil {
-		return nil, err
-	}
-	return m.db.TrialDetailsRaw(args.TrialID)
```

Comment:
is this in use anywhere else?

Reply:
nope, this part is good to delete; this whole file should go away in a future PR once we update the other checks in e2e_tests